src/erlando.app.src

```diff
@@ -1,6 +1,17 @@
 {application, erlando,
  [{description, "Syntax extensions for Erlang"},
-  {vsn, "%%VSN%%"},
+  {vsn, git},
   {modules, []},
   {registered, []},
-  {applications, []}]}.
+  {applications, []},
+  {pkg_name, "rabbitmq_erlando"},
```

Comment:
If you are interested in maintaining Erlando I suggest that you rename the package to `erlando2` and change the version to `2.0` to indicate a new age for this project. Otherwise I'm afraid we cannot stamp this package with "rabbitmq" since team RabbitMQ neither uses nor maintains this project (and it has been the case for years).